### PR TITLE
SDK - Components - Fixed YAML formatting for some components

### PR DESCRIPTION
**Pull request by Ark-kun**
This fixes formatting for components where function does not have a return annotation.
The low-level cause of issue: Trailing whitespace when there are no serializers.
Trailing whitespace triggers ugly YAML string formatting.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2529)

### CODE REVIEW

File: sdk/python/kfp/components/_python_op.py [hunk 1]
```diff
@@ -483,6 +483,8 @@ def get_serializer_and_register_definitions(type_name) -> str:
         '_output_files = _parsed_args.pop("_output_paths", [])',
     ])
 
+    output_serialization_code = ''.join('    ' + s + ',' + '\n' for s in output_serialization_expression_strings)
```
Comment (numerology) [inline sdk/python/kfp/components/_python_op.py]: nit: use '    {},\n'.format(s) to prevent the string construction from running 4 times.

Comment (Ark-kun) [reply sdk/python/kfp/components/_python_op.py]: Done. Although I'm not sure it's more efficient due to the need to parse the template string and construct the new string from parts.

**Pull request merged**